ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestDetailsPage.tsx

```diff
@@ -132,12 +142,15 @@ function ExceptionRequestDetailsPage() {
         );
     }
 
-    const { status, cves, scope } = vulnerabilityException;
+    const { status, cves, scope, requester } = vulnerabilityException;
 
     const isApprovedPendingUpdate = status === 'APPROVED_PENDING_UPDATE';
     const showApproveDenyButtons =
         hasWriteAccessForApproving &&
         (status === 'PENDING' || status === 'APPROVED_PENDING_UPDATE');
+    const showCancelButton =
+        currentUser.userId === requester.id &&
+        (status === 'PENDING' || status === 'APPROVED_PENDING_UPDATE');
```

Comment:
I think admins will have the ability to cancel too. Would adding a check for `hasWriteAccessForApproving` here cover that case?

```ts
    const showCancelButton =
        (hasWriteAccessForApproving || currentUser.userId === requester.id) &&
        (status === 'PENDING' || status === 'APPROVED_PENDING_UPDATE');
```

Reply:
Looks like they only want the requestor to be able to cancel the request

> Only the one who created a request can cancel it; "A requester should be able to cancel (purge) the deferral and false positive request at any time (pending, approved or declined)"

https://docs.google.com/document/d/12hYUp1FI070syioAG_iep-ZbMdi4hZwgprndh8azcDg/edit?usp=sharing

Reply:
I'll make the change to the conditional to show up during those phases

Reply:
Ah fair enough, I saw the opposite comment on one of the mocks but the requirements doc should take precedence. 👍